charts/timesketch/Chart.lock

```diff
@@ -0,0 +1,12 @@
+dependencies:
+- name: postgresql
+  repository: https://charts.bitnami.com/bitnami
+  version: 12.2.7
+- name: redis
+  repository: https://charts.bitnami.com/bitnami
+  version: 17.9.3
+- name: opensearch
+  repository: https://opensearch-project.github.io/helm-charts/
+  version: 2.11.3
```

Comment:
This will install opensearch 2.x - we don't fully support that yet. Does it work correctly?

Reply:
I was able to process a timeline and see opensearch index the results. Is there anything I can test for compatibility?